pkg/dynamicinstrumentation/codegen/c/expressions.h

```diff
@@ -0,0 +1,309 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
```

Comment:
Please remove the Apache license from any eBPF source files, since they are usually GPL licensed